### PR TITLE
Iosxe/show ap rf profile summary

### PR DESCRIPTION
**Pull request by reustaquio22**
Parser for Iosxe/show ap rf profile summary

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+
+# ===============================
+# Schema for:
+#  * 'show ap rf-profile summary'
+# ===============================
+class ShowApRfProfileSummarySchema(MetaParser):
+    """Schema for show ap rf-profile summary."""
+
+    schema = {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/show_ap.py:14]: schema looks good

**Comment by karmoham** [comment]
Thanks, Roy for the contribution!

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        # Number of RF-profiles: 14
+        rf_profile_count_capture = re.compile(r"^Number\s+of\s+RF-profiles:\s+(?P<rf_profile_count>\d+)")
+        # Custom-RF_a                      5 GHz    Custom-RF_a_Desc                     Up
+        rf_profile_info_capture = re.compile(
+            r"^(?P<rf_profile_name>\S+)\s+(?P<band>\S+\s+\S+)\s+(?P<description>.*)(?P<state>(Up|Down))")
+
+        remove_lines = ('RF Profile', '----')
+
+        # Remove unwanted lines from raw text
+        def filter_lines(raw_output, remove_lines):
```
[the remark this answers is not in the document]
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_ap.py]: @karmoham, This was updated

File: changelog/undistributed.rst [hunk 3]
```diff
@@ -0,0 +1,19 @@
+* Please follow the template we introduced in NOVEMBER.md file.
+* Every parser need to be added under the corresponding feature.
+
+| Module                  | Version       |
+| ------------------------|:-------------:|
+| ``genie.libs.parser``   |               |
+
+--------------------------------------------------------------------------------
```
Comment (karmoham) [inline changelog/undistributed.rst:8]: Can you please fix the conflict in the undistributed file?

Comment (karmoham) [reply changelog/undistributed.rst:8]: This is the current dev branch code https://github.com/CiscoTestAutomation/genieparser/blob/master/changelog/undistributed.rst

Comment (reustaquio22) [reply changelog/undistributed.rst]: @karmoham, conflicts have been resolved.

**Comment by karmoham** [comment]
Thanks Roy, looking forward to the next pull request :)

**Pull request merged**